### PR TITLE
[DEC-2134] - Update existing markets with finalized config from research

### PR DESCRIPTION
**Pull request by clemire**
According to [this sheet](https://docs.google.com/spreadsheets/d/1QmuwS0LljTrZmkFiYMhUR9MfFa4jhgfzhgDC9z0v2LQ/edit#gid=60914733). Addition of DYDX token in [this PR](https://github.com/dydxprotocol/v4-chain/pull/466). 

Deprecated PR which combines both [here](https://github.com/dydxprotocol/v4-chain/pull/455).

### Summary by CodeRabbit

- Refactor: Updated the configuration of market tickers across various exchanges in the testnet environment. This includes additions, removals, and modifications to ensure accurate representation of current market conditions.
- New Feature: Enhanced the price feed daemon with updated cryptocurrency-to-USD values for improved accuracy in testing scenarios.

### CODE REVIEW

File: protocol/daemons/pricefeed/client/constants/testdata/bch_exchange_config.json [hunk 1]
```diff
@@ -6,18 +6,14 @@
...
     {
-      "exchangeName": "Bitstamp",
-      "ticker": "BCH/USD"
+      "exchangeName": "Bybit",
+      "ticker": "BCHUSDT",
+      "adjustByMarket": "USDT-USD"
     },
     {
       "exchangeName": "CoinbasePro",
       "ticker": "BCH-USD"
```
Comment (ttl33) [inline protocol/daemons/pricefeed/client/constants/testdata/bch_exchange_config.json:15]: I see the following, which suggests that we were supposed to update. Was this already our setting before research's suggestion?


Screenshot 2023-10-04 at 3 45 07 PM

```
Coinbase_Pro |   | BCH | USDT | Remove
Coinbase_Pro |   | BCH | USD | Add


```

Comment (ttl33) [reply protocol/daemons/pricefeed/client/constants/testdata/bch_exchange_config.json:15]: Similarly for `Kraken`

**Review by yang-dydx — APPROVED** [review]
thanks!

File: protocol/daemons/pricefeed/client/constants/testdata/bch_exchange_config.json (hunk 1 above)
Comment (clemire) [reply protocol/daemons/pricefeed/client/constants/testdata/bch_exchange_config.json:15]: Yes, they gave us some updates that were already applied.

Comment (clemire) [reply protocol/daemons/pricefeed/client/constants/testdata/bch_exchange_config.json:15]: Thank you for the close reading!

Comment (clemire) [reply protocol/daemons/pricefeed/client/constants/testdata/bch_exchange_config.json:15]: If you check the "price diff oracles progress" page, you'll see where I marked some updates were already applied.

**Review by ttl33 — APPROVED** [review]

**Pull request merged**